moveos/moveos/src/moveos.rs

```diff
@@ -39,6 +42,7 @@ pub struct TransactionOutput {
     /// The new state root after the transaction execution.
     pub state_root: H256,
     pub status: KeptVMStatus,
+    pub status_reexplain: VmStatusExplainView,
```

Comment:
VmStatusExplainView should not be part of TransactionOutput. But we can refactor in #239